console/console-init/ui/src/Components/AddressDetail/CreateAddressDefinition.tsx

```diff
@@ -91,8 +91,8 @@ export const AddressDefinitaion: React.FunctionComponent<IAddressDefinition> = (
       if(plan.Spec.AddressType === type){
         return {
           value: plan.Spec.DisplayName,
-          label: plan.Spec.DisplayName,
-          disabled: false
+          label: plan.Spec.DisplayName.charAt(0).toUpperCase() + plan.Spec.DisplayName.slice(1),
```

Comment:
Wouldn't a `text-transform: capitalize;` css rule be better instead of this?

Reply:
Will be way better! I am updating my PR.